content/docs/guides/converting-wordpress-to-tina.mdx

```diff
@@ -0,0 +1,218 @@
+---
+title: Migrating WordPress Content
+last_edited: '2024-11-08T01:49:52.552Z'
+next: ''
+previous: ''
+---
+
+## Introduction
+
+This guide is a walkthrough using a **WordPress to Markdown converter**, which is recommended for most use cases where you want to migrate your content from WordPress to Tina.
+
+This guide focuses on transferring content and media only and will not include page styling.
+
+Other conversion options that are not covered in this guide include:
+
+- Using the WordPress Rest API - best for extracting the pure html for page or post content in JSON format
+- Using the WPGraphQL plugin - best for cases where you want to query and extract specific data and content
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org/) is installed
+- You have set up a [Tina starter](https://tina.io/docs/introduction/using-starter)
```

Comment:
i think basic understanding of Tina is also a prerequisties, maybe a link to where to learn Tina

Reply:
Done - added prerequisite to have a basic understanding of the editor and the schema, with links to them